tests/BoldContentTest.php

```diff
@@ -4,7 +4,15 @@
 class BoldContentTest extends DeltaTestCase
 {
     public $json = <<<'JSON'
-{"ops":[{"attributes":{"bold":true},"insert":"Formatted"},{"insert":"\n"},{"attributes":{"bold":true},"insert":"Heading 1"},{"attributes":{"header":1},"insert":"\n"},{"insert":"list"},{"attributes":{"list":"bullet"},"insert":"\n"},{"insert":"list"},{"attributes":{"list":"bullet"},"insert":"\n"},{"insert":"Heading 2"},{"attributes":{"header":2},"insert":"\n"}]}
```

Comment:
Don't you want to keep the 'Formatted' bold part before to show that still works?